backstop/tests/checkbox.html

```diff
@@ -167,99 +109,39 @@ <h2>Default</h2>
       <div class="iui-input-container iui-with-message iui-disabled">
         <div class="iui-label">Checkbox group label</div>
         <div class="iui-input-group">
-          <label class="iui-checkbox">
+          <label class="iui-checkbox-wrapper iui-disabled">
```

Comment:
This one is an important difference. The wrapper explicitly needs `iui-disabled` now because the checkbox-label does not rely on `input:disabled` anymore. Same thing in radio.

Reply:
That is fine, #465 is [doing the same](https://github.com/iTwin/iTwinUI/blob/977bcee9873dccb2c9b4732325918d864f386dc2/src/toggle-switch/toggle-switch.scss#L23).  I will copy your class name structure for toggle switch as well (`.iui-toggle-switch-wrapper` & `.iui-toggle-switch`).